File: scratch/fix.js
const fs = require('fs');
const lines = fs.readFileSync('D:/000Memorade/index.html', 'utf8').split('\n');

const replacement = [
    '      const rawCats = getCategories(langPair);',
    '      const levels = [',
    '        ...(rawCats.core || []),',
    '        ...(rawCats.sentences || []),',
    '        ...(rawCats.special || []),',
    '        ...(rawCats.yksHazirlik || []),',
    '        ...(rawCats.ydsKelimeleri || [])',
    '      ];'
];

lines.splice(1464, 10, ...replacement);
fs.writeFileSync('D:/000Memorade/index.html', lines.join('\n'), 'utf8');

File: scratch/fix.py
with open('D:/000Memorade/index.html', 'r', encoding='utf-8') as f:
    lines = f.readlines()

lines[1464:1474] = [
    '      const rawCats = getCategories(langPair);\n',
    '      const levels = [\n',
    '        ...(rawCats.core || []),\n',
    '        ...(rawCats.sentences || []),\n',
    '        ...(rawCats.special || []),\n',
    '        ...(rawCats.yksHazirlik || []),\n',
    '        ...(rawCats.ydsKelimeleri || [])\n',
    '      ];\n'
]

with open('D:/000Memorade/index.html', 'w', encoding='utf-8') as f:
    f.writelines(lines)
